plugins/flytekit-duckdb/flytekitplugins/duckdb/task.py

```diff
@@ -10,6 +12,25 @@
 pa = lazy_module("pyarrow")
 
 
+class MissingSecretError(ValueError):
```

Comment:
can't we move this to the test module since this class isn't being used anywhere?